colorscad.sh

```diff
@@ -280,13 +282,9 @@ if [ "$FORMAT" = amf ]; then
 		id=0
 		IFS=$'\n'
 		for COLOR in $COLORS; do
-			IFS=','; set -- $COLOR
-			R=${1#[}
-			G=$2
-			B=$3
-			A=${4%]}
+		  IFS=, read -r R G B A <<<"${COLOR//[\[\] ]/}"
 			echo " <material id=\"${id}\"><color><r>${R// }</r><g>${G// }</g><b>${B// }</b><a>${A// }</a></color></material>"
```

Comment:
The space-strippling below should not be needed anymore, we can improve readability now
```suggestion
			echo " <material id=\"${id}\"><color><r>${R}</r><g>${G}</g><b>${B}</b><a>${A}</a></color></material>"
```